src/services/congressperson.service.js

```diff
@@ -93,6 +93,8 @@ module.exports = function setupCongresistaService({
         : {};
       const congresspersonDetail = await CongresspersonModel.findOne({
         where,
+        benchmack: true,
```

Comment:
Es una opción válida en sequelize? Y si es así, tal vez debería ser `benchmark`?

Reply:
sí lo he probado con esa opción y se encuentra también en la documentación: https://sequelize.org/v4/class/lib/sequelize.js~Sequelize.html 
![image](https://user-images.githubusercontent.com/6945270/154356553-79a5cdd5-f8ff-4003-b865-a512c6e4f99f.png)

Reply:
Y lo llegaste a probar? porque estamos utilizando la v6
https://github.com/openpolitica/open-tracker-backend/blob/881f570a90ad6c62913ef046585b9a54b9f1ebbc/package.json#L32
Pregunto porque no sé si sequelize tiene opciones retroactivas

Reply:
sí lo probé en mi computadora y sí me ha funcionado

Reply:
sino normal se elimina, solo era para obtener el tiempo de los queries, para tener alguna referencia de los tiempos

Reply:
Sí esta también en la versión actual: https://sequelize.org/v6/class/src/sequelize.js~Sequelize.html

Reply:
Fix in #[9a92c19](https://github.com/openpolitica/open-tracker-backend/pull/55/commits/9a92c19d414ddd0a0ff8e6fe2e9f252539ab734d)